src/web/routes/application/select-address/select-address.js

```diff
@@ -7,11 +7,17 @@ const pageContent = ({ translate }) => ({
   enterManualAddress: translate('buttons:enterManualAddress'),
   explanation: translate('address.explanation'),
   change: translate('change'),
-  continue: translate('buttons:continue')
+  continue: translate('buttons:continue'),
+  selectAddressLabel: translate('address.selectAddressLabel')
+})
+
+const buildAddressOption = result => ({
```

Comment:
does this mean we're not storing the individual fields, just the concatenated address?